database/migrations/2021_04_07_165323_create_order_financings_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateOrderFinancingsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('order_financings', function (Blueprint $table) {
+            $table->id();
+            $table->integer('user_id');
+            $table->integer('lender_organization_id');
+            $table->integer('payment_period');
+            $table->softDeletes();
+            $table->timestamps();
+        });
```

Comment:
It will be nice to have the percentage also here

Reply:
Well noted Boss... Will add it